gandalf.js

```diff
@@ -1,6 +1,6 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!